src/Auctions/AuctionFactory.sol

```diff
@@ -0,0 +1,194 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.8.18;
+
+import {ERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+
+import {Clonable} from "../utils/Clonable.sol";
+import {Auction} from "./Auction.sol";
+
+/// @title AuctionFactory
+/// @notice Deploy a new Auction.
+contract AuctionFactory is Clonable {
```

Comment:
Would be nice to have a auction list like the ajna [ERC20PoolFactory](0x6146DD43C5622bB6D12A5240ab9CF4de14eDC625)